sources/packages/web/src/components/aest/AssessApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <v-form ref="assessApplicationOfferingChangeRequestForm">
+    <modal-dialog-base
+      :showDialog="showDialog"
+      :title="getTitle"
+      :max-width="780"
+    >
+      <template #content
+        ><error-summary
+          :errors="assessApplicationOfferingChangeRequestForm.errors"
+        />
+        <p class="my-4">
+          Outline the reasoning for
+          <span
+            v-if="
+              showParameter ===
+              ApplicationOfferingChangeRequestStatus.DeclinedBySABC
+            "
+            >declining</span
+          ><span v-else>approving</span>
+          this request. Please add the application number.
+        </p>
+        <p class="label-bold mb-1">Notes</p>
```

Comment:
Figma is not targeting Vuetify. For the Vuetify component, we do not create extra labels we just use the out-of-box component `label` property even if it is a bit different from Figma.
https://github.com/bcgov/SIMS/blob/cf55ff4172c2dd87de6e44b6ee523084062a6e7f/sources/packages/web/src/components/aest/institution/modals/AssessOfferingModal.vue#L9-L15